proposed/extended-coding-style-guide.md

```diff
@@ -145,6 +145,9 @@ When the opening `<?php` tag is on the first line of the file, it MUST be on its
 own line with no other statements unless it is a file containing markup outside of PHP
 opening and closing tags.
 
+Use declarations [Import statements] MUST never begin with a leading backslash as they
```

Comment:
Looks a bit weird to me. Either it should say "Use declarations" or "Import statements". Not both at the same time.